src/Intervals.jl

```diff
@@ -1,6 +1,8 @@
 module Intervals
 
 using Dates
+using Infinity
```

Comment:
Is this needed as you explicitly import methods below?

Reply:
If you don’t then you can’t use the infinity symbol correctly. I suppose I could just import it with the other items.

Reply:
In the context of the package I don't think we're referencing the infinity symbol

Reply:
hmm good point I guess it's just for the tests.